R/Tables.R

```diff
@@ -1,151 +1,152 @@
+TADA_SummarizeCensoredData <- function(.data, spec_cols = c("TADA.CharacteristicName", "TADA.ResultMeasure.MeasureUnitCode", "TADA.ResultSampleFractionText", "TADA.MethodSpecificationName")) {
+  if (any(is.na(.data$TADA.ResultMeasureValue))) {
     warning("Dataset contains data missing both a result value and a detection limit. Suggest removing or handling. See TADA Harmonization vignette for an example.")
   }
-  
-  if(!"TADA.CensoredData.Flag"%in%names(.data)){
-    cens = TADA_IDCensoredData(.data)
-  }else{
-    cens = .data
+
+  if (!"TADA.CensoredData.Flag" %in% names(.data)) {
+    cens <- TADA_IDCensoredData(.data)
+  } else {
+    cens <- .data
   }
-  
-  sum_low = cens%>%dplyr::group_by_at(spec_cols)%>%
-    dplyr::filter(TADA.CensoredData.Flag%in%c("Non-Detect", "Uncensored"))%>%
-    dplyr::summarise(Measurement_Count = length(unique(ResultIdentifier)), Censored_Count = length(TADA.CensoredData.Flag[TADA.CensoredData.Flag=="Non-Detect"]), Percent_Censored = length(TADA.CensoredData.Flag[TADA.CensoredData.Flag=="Non-Detect"])/length(TADA.CensoredData.Flag)*100, Censoring_Levels = length(unique(TADA.ResultMeasureValue[TADA.CensoredData.Flag=="Non-Detect"])))%>%
-    dplyr::filter(Censored_Count>0)%>%
+
+  sum_low <- cens %>%
+    dplyr::group_by_at(spec_cols) %>%
+    dplyr::filter(TADA.CensoredData.Flag %in% c("Non-Detect", "Uncensored")) %>%
+    dplyr::summarise(Measurement_Count = length(unique(ResultIdentifier)), Censored_Count = length(TADA.CensoredData.Flag[TADA.CensoredData.Flag == "Non-Detect"]), Percent_Censored = length(TADA.CensoredData.Flag[TADA.CensoredData.Flag == "Non-Detect"]) / length(TADA.CensoredData.Flag) * 100, Censoring_Levels = length(unique(TADA.ResultMeasureValue[TADA.CensoredData.Flag == "Non-Detect"]))) %>%
+    dplyr::filter(Censored_Count > 0) %>%
     dplyr::mutate("TADA.CensoredData.Flag" = "Non-Detect")
-  
-  sum_hi = cens%>%dplyr::group_by_at(spec_cols)%>%
-    dplyr::filter(TADA.CensoredData.Flag%in%c("Over-Detect", "Uncensored"))%>%
-    dplyr::summarise(Measurement_Count = length(unique(ResultIdentifier)), Censored_Count = length(TADA.CensoredData.Flag[TADA.CensoredData.Flag=="Over-Detect"]), Percent_Censored = length(TADA.CensoredData.Flag[TADA.CensoredData.Flag=="Over-Detect"])/length(TADA.CensoredData.Flag)*100, Censoring_Levels = length(unique(TADA.ResultMeasureValue[TADA.CensoredData.Flag=="Over-Detect"])))%>%
-    dplyr::filter(Censored_Count>0)%>%
+
+  sum_hi <- cens %>%
+    dplyr::group_by_at(spec_cols) %>%
+    dplyr::filter(TADA.CensoredData.Flag %in% c("Over-Detect", "Uncensored")) %>%
+    dplyr::summarise(Measurement_Count = length(unique(ResultIdentifier)), Censored_Count = length(TADA.CensoredData.Flag[TADA.CensoredData.Flag == "Over-Detect"]), Percent_Censored = length(TADA.CensoredData.Flag[TADA.CensoredData.Flag == "Over-Detect"]) / length(TADA.CensoredData.Flag) * 100, Censoring_Levels = length(unique(TADA.ResultMeasureValue[TADA.CensoredData.Flag == "Over-Detect"]))) %>%
```

Comment:
This line is long, it was before too, just tells us the styler isn't enforcing line char limit

Reply:
Oooh good point--something to look into. It looks like this comparison view is trying to ensure each line is the same width on the screen...I think this is indicating that styler added a line after cens%>% on line 132, which moves everything else down. There appears to be a big gap on line 134 on the "after" pane because the "before" code made line 134 longer, and in the "after" it's all on line 135.